utils/nsutils/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2016 Rafał Miłecki <rafal@milecki.pl>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nsutils
+PKG_VERSION:=0.2
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/rd235/nsutils.git
+PKG_SOURCE_DATE:=2022-02-27
+PKG_SOURCE_VERSION:=d70867d11d455c19aece0c75791dae5c40b88ca6
+PKG_MIRROR_HASH:=94b33cde9240bad9803e2c796437d59c57405e3da95cdb762c9ba8fe041e9643
+
+PKG_FIXUP:=autoreconf
+PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/nsutils
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Linux namespace utilities
+  DEPENDS:=+libbsd +libcap
+  MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
```

Comment:
Redundant.

Reply:
Maintainer is in wrong place :) Sorry :)